docs/index.html

```diff
@@ -93,8 +93,7 @@ <h1 class="desc-h1">Nighthawk and Meshery</h1>
       </div>
     <div class="nighthawk-meshery-div">
       <div class="container description-div description-div-grpc">
-        <div class="custom-desc-div">
-    
+        <div class="">
```

Comment:
@akshitarora921 why did you removed the class??

Reply:
It was not doing anything as it only have one child tag, so `display:flex` or `flex-direction: column-reverse` were not making any impact.